eng/CodeCoverage.runsettings

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<RunSettings>
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <DataCollector friendlyName="XPlat Code Coverage" enabled="true">
+        <Configuration>
+          <Format>cobertura</Format>
+          <ExcludeByAttribute>ExcludeFromCodeCoverageAttribute,GeneratedCodeAttribute,Obsolete</ExcludeByAttribute>
+          <IncludeTestAssembly>false</IncludeTestAssembly>
```

Comment:
Why do we have this set to false? I thought excluding the test assembly is desiriable.

Reply:
That's why it's false, i.e. don't *include* the test assembly. 😉